inst/ms/aggreCAT.qmd

```diff
@@ -400,39 +423,43 @@ for a single claim to illustrate this structure of the core
 `data_ratings` dataset.
 
 ```{r}
-#| label: data_ratings
+#| label: data_ratings-sample
 #| message: false
 #| prompt: true
+#| results: hold
+
 library(tidyverse,quietly = TRUE)
 library(aggreCAT)
 data(data_ratings)
```

Comment:
This line is superfluous, `data_ratings` is loaded with the package.

Reply:
OK, I've deleted `data(data_ratings)` and namespaced `data_ratings` in the next line to communicate to the reader that `data_ratings` is an export of aggreCAT package.